siliconcompiler/schema/schema_obj.py

```diff
@@ -591,6 +638,26 @@ def _keypath_empty(self, key):
 
         return value_empty
 
+    ###########################################################################
+    def get_history(self, job):
```

Comment:
Is this meant to be a public function. Can we rename to 'history()' or 'getschema()'  We currently have {get, getkeys,  getdict}